contrib/oot.txt

```diff
@@ -1,7 +1,12 @@
 # List of out-of-tree repositories
 # Format: repo name
 https://github.com/R4PaSs/brewnit.git brewnit
+https://github.com/Morriar/CatFacts.git catfacts
```

Comment:
I made this one yesterday for a lab, maybe it's not relevant in OOT? As the code is dirty/undocumented you see?